history/provinces/00_k_black_morass.txt

```diff
@@ -18,6 +18,7 @@
 	589.11.1={
 		culture=dalaranian
 		religion=kirin_tor
+		holding = castle_holding
```

Comment:
this would be a good place to add random buildings because of the 14 year gap to the next bookmark, or add whatever buildings you feel would be appropriate! caster, air, etc